src/assets/locales/en/translation.json

```diff
@@ -0,0 +1,18 @@
+{
+  "movie_format": "formats",
+  "movie_all": "all",
```

Comment:
you can separate your translations by page/component
for example
{
"films": {
"filter": {
"title": "test"
}
}
}

And you can call by films.filter.title

Reply:
the folder locales should be in /src/locales

Reply:
done